src/reducers/scrollStateHelper.js

```diff
@@ -279,94 +279,14 @@ export function updateViewHeight(state, {
 
 /**
  * @param {!Object} state
- * @param {number} deltaY
- */
-export function scrollBy(state, deltaY) {
```

Comment:
@KamranAsif @miskreant pretty sure I had a detailed comment here which got blown away by me merging v0.8 in (stupid github).  Could one of you check your email and see if you have a notification with the text of the comment?